getting_started/setup_linux/setup_ktest_environment.rst

```diff
@@ -58,16 +58,24 @@ Set up a target
 	 sudo emacs /etc/default/grub
 
    b) Change ``GRUB_DEFAULT=[n]`` to ``GRUB_DEFAULT=saved``.
-      
-   c) Add ``GRUB_DISABLE_SUBMENU=y`` to the end of the file and save it.
 
-      This step is necessary because submenus confuse ktest.
+   c) You must disable GRUB submenus because they confuse ktest:
+
+      .. code-block:: console
+
+	echo 'GRUB_DISABLE_SUBMENU=y' | sudo tee -a /etc/default/grub.d/disable-submenu.cfg
 
    d) Update the grub configuration.
 
       .. code-block:: console
-	 
-	 sudo update-grub
+
+	# Better safe than _very_ sorry
+	sudo cp /boot/grub/grub.cfg /boot/grub/saved_grub.cfg
+
+	sudo update-grub
```

Comment:
Note: this sometimes can cause un-bootable system which can be only recovered by booting to a live env, chrooting to the existing linux install and `grub-install --target=x86_64-efi ....`

Reply:
I tried to chroot and run update-grub inside the chroot but that did not work, it failed with: `can't find device for /, is /dev mounted?`
GRUB was still installed on my system, that wasn't my problem.

What worked _for me_ was to find and manually enter grub commands and parameters at boot time. This let me boot. Then I disabled the submenus and ran `update-grub` from the "real" system. 

Different OS and different situations will require different solutions. Rescuing unbootable systems is a very complex topic; we can't provide bullet proof advice in such a limited space here.

Reply:
> I tried to chroot and run update-grub inside the chroot but that did not work, it failed with: `can't find device for /, is /dev mounted?` GRUB was still installed fine on my system in that instance, that wasn't my problem.

Few things should be done to be able to update grub in chroot, something like:
https://wiki.gentoo.org/wiki/Chroot#Configuration

> What worked _for me_ was to find and manually enter grub commands and parameters at boot time.

Wow

> This let me boot. Then I disabled the submenus and ran `update-grub` from the "real" system.
> 
> Different OS and different situations will require different solutions. Rescuing unbootable systems is a very complex topic; we can't provide bullet proof advice in such a limited space here.

Sure!

Reply:
@marc-hb Are you ready for this PR to be merged? Thanks.

Reply:
Yes please merge.

Reply:
@marc-hb Thanks!